model/kubernetes/kubernetes_cluster.rb

```diff
@@ -8,6 +8,7 @@ class KubernetesCluster < Sequel::Model
   many_to_one :private_subnet
   many_to_one :project
   many_to_many :cp_vms, join_table: :kubernetes_clusters_cp_vms, class: :Vm, order: :created_at
+  one_to_many :kubernetes_nodepools
```

Comment:
Since both of them are in the `Kubernetes` context, `nodepools` might be more readable. 
I prefer `kubernetes_cluster.nodepools` instead of `kubernetes_cluster.kubernetes_nodepools`

similar discussion: https://github.com/ubicloud/ubicloud/pull/2628#discussion_r1920043782